Formula/i/intercept.rb

```diff
@@ -1,8 +1,8 @@
 class Intercept < Formula
   desc "Static Application Security Testing (SAST) tool"
   homepage "https://intercept.cc"
-  url "https://github.com/xfhg/intercept/archive/refs/tags/v1.6.1.tar.gz"
-  sha256 "76bfd38b940702a3937712a8b66623c7b36f7e681e4f73f7c9778b9083621b99"
+  url "https://github.com/xfhg/intercept/archive/refs/tags/v1.0.12.tar.gz"
+  sha256 "2732a3e895a9685ba6f112e7e372627aebfa340a94bf4716462b382075593308"
   license "AGPL-3.0-only"
```

Comment:
```suggestion
  license "EUPL-1.2"
  version_scheme 1
```

The license is now EUPL-1.2 with this iteration: https://github.com/xfhg/intercept/blob/edf8c94648abd8278b6549b6544468f6dedc0fce/LICENSE.md

It seemed to me like upstream has kind of restarted their versions from 1.0.3, so I agree that bumping the `version_scheme` makes sense.